cmake/projects/geos/hunter.cmake

```diff
@@ -1,9 +1,14 @@
 # Copyright (c) 2015, Ruslan Baratov, Alexandre Pretyman
 # All rights reserved.
 
-# !!! DO NOT PLACE HEADER GUARDS HERE !!!
```

Comment:
ooops, sorry. I don't remember touching it, something must have gone wrong on my side